lib/analytics.js

```diff
@@ -30,6 +31,7 @@ module.exports = class Analytics {
     batchInterval = getBatchInterval()
   }) {
     this._propertySettings = propertySettings
+    this._trackAnalytics = trackAnalytics
```

Comment:
I don't think you need this, or the one above, as the decision on whether to track is taking in index.js rather than here